src/index.ts

```diff
@@ -189,15 +189,32 @@ const server = createServer(async (request, response) => {
       };
       const event = createReferencesEvent([sseData]);
       response.write(event);
-    }
 
-    // We should keep all optional parameters out of this call, so it can work for any model (in case we've
-    // just run the execute-model tool).
-    const stream = await modelsAPI.inference.chat.completions.create({
-      model: functionCallRes.model,
-      messages: functionCallRes.messages,
-      stream: true,
-    });
+      // We should keep all optional parameters out of this call, so it can work for any model (in case we've
+      // just run the execute-model tool).
+      if (!["o1-mini", "o1-preview"].includes(args.model)) {
```

Comment:
/nitpick - Can we swap the if/else chunks so we're not doing an `if` on a negated conditional?

```suggestion
      if (["o1-mini", "o1-preview"].includes(args.model)) {
```

Reply:
Excellent idea.